src/opnsense/www/themes/opnsense/assets/stylesheets/main.scss

```diff
@@ -884,3 +884,20 @@ div[data-for*="help_for"] {
 .dashboard_grid_column table {
   table-layout: fixed;
 }
+
+$ipsec_bright_color: #EEEEEE;
+$ipsec_dark_color: #777777;
+
+#ipsec {
+  #ipsec-mobile, #ipsec-tunnel, #ipsec-Overview {
```

Comment:
I'd prefer this to be lowercase all the way

Reply:
me too but the ids aren't mine ;)

Reply:
but you touch the same lines so check mate! :P